djpaddle/views.py

```diff
@@ -51,7 +51,7 @@ def post(self, request, *args, **kwargs):
 
         if alert_name in self.SUPPORTED_WEBHOOKS:
             signal = getattr(signals, alert_name, None)
-            if signal is not None:
+            if signal is not None:  # pragma: no cover
```

Comment:
I added `pragma: no cover` here as I wasn't sure of a way to get here via a test without making a change to either `signals.py` or `SUPPORTED_WEBHOOKS` as they currently have matching names (which they should).